tests/Integration/AppCheckTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kreait\Firebase\Tests\Integration;
+
+use Kreait\Firebase\AppCheck\AppCheckToken;
+use Kreait\Firebase\AppCheck\VerifyAppCheckTokenResponse;
+use Kreait\Firebase\Contract\AppCheck;
+use Kreait\Firebase\Tests\IntegrationTestCase;
+
+/**
+ * @internal
+ *
+ * @group database-emulator
+ * @group emulator
+ */
+final class AppCheckTest extends IntegrationTestCase
+{
+    public AppCheck $appCheck;
+    public string $appId;
+
+    protected function setUp(): void
+    {
+        $this->appCheck = self::$factory->createAppCheck();
+        $this->appId = 'com.example.test-app';
```

Comment:
How did you make this work when running the integration tests?

Reply:
Apparently I didn't have them running. I didn't realize the integration tests were being auto-skipped locally.

I've now created a test project and added the necessary `test_credentials.json` and `test_rtdb.json` files and some of the integration tests are passing but I'm not sure what I need to put in `test_devices.json` to get the rest passing? What exactly are the registration tokens here?

Reply:
If you run only the AppCheck tests (`phpunit tests/Integration/AppCheckTest.php`), you might not need the FCM registration tokens. I generate one with https://github.com/firebase/quickstart-js/tree/master/messaging and then copy it as a JSON array into the `test_devices.json` file (`["<device token>"]`).

That's a good reminder to add a `CONTRIBUTING.md` 😅